app/assets/javascripts/govuk_publishing_components/lib/cookie-functions.js

```diff
@@ -207,14 +211,23 @@
     }
   }
 
+  window.GOVUK.cookieCategories = function () {
+    if (window.GOVUK.overrideCookieCategories) {
+      return window.GOVUK.overrideCookieCategories()
+    } else {
```

Comment:
We probably need some kind of documentation on how to do this?

Reply:
Yeah, good point... Not sure where the best place for that would be. I could add a new file to `/docs`?

Reply:
Or perhaps to the component guide, as it's related to the cookie banner component.

Reply:
I'd add it wherever there is similar documentation and link to that from the component guide, with a summary explanation.

Reply:
@andysellick Added some documentation on [using the cookie banner](https://github.com/alphagov/govuk_publishing_components/blob/8eeecba8ee89fbf27ebbe55170ed7a3d6adbc30a/docs/using-cookie-banner.md) and linked to this from the component guide.